build/generateImplicitAPI.js

```diff
@@ -104,8 +104,11 @@ function createImplicitMethods(){
     const exchanges = Object.keys(storedCamelCaseMethods);
     for (const index in exchanges) {
         const exchange = exchanges[index];
-        const camelCaseMethods = storedCamelCaseMethods[exchange];
-        const underscoreMethods = storedUnderscoreMethods[exchange]
+        let camelCaseMethods = storedCamelCaseMethods[exchange];
+        let underscoreMethods = storedUnderscoreMethods[exchange];
+        // remove duplicates
+        camelCaseMethods = [... new Set(camelCaseMethods)];
```

Comment:
@ttodua I'm not sure this is the right place to fix the issue, maybe we should not have duplicated endpoints in exchanges that are parent<->child?

Reply:
yeah right, seems i've just applied "painkiller" instead of solution. let me fix it better

Reply:
issue was in other place, this script didnt need modification.
Now PR is ready, whenever you can review.